powersimdata/data_access/data_access.py

```diff
@@ -177,14 +158,14 @@ def checksum(self, relative_path):
         :param str relative_path: path relative to root
         :return: (*str*) -- the checksum of the file
         """
-        return "dummy_value"
+        return self.fs.hash(relative_path, "sha256")
```

Comment:
Did we really have `"dummy_value"` running in prod?

Reply:
Yeah.. although we override by using `sha1sum` if using the server. The dummy value was only used for local/container environments.